src/picongpu/include/particles/Particles.tpp

```diff
@@ -72,7 +73,9 @@ datasetID( datasetID )
 
     log<picLog::MEMORY > ( "size for all exchange = %1% MiB" ) % ( (double) sizeOfExchanges / 1024. / 1024. );
 
-    const uint32_t commTag = FrameType::CommunicationTag + SPECIES_FIRSTTAG;
+    const uint32_t commTag = PMacc::traits::GetUniqueTypeId<FrameType>::uid + SPECIES_FIRSTTAG;
+    log<picLog::CRITICAL > ( "communication tag for species %1%: %2%" ) % FrameType::getName( ) % commTag;
```

Comment:
let's use `MEMORY` here, communication is just a implementation aspect of our memory in libPMacc and comes closest ;)